Jenkinsfile

```
@@ -139,6 +141,16 @@ try {
         stash name: "rpms-alma8", includes: 'output/noarch/*.rpm'
         sh 'rm -rf output'      
       }
+    },
+    'Debian bullseye packaging and signing': {
+      node {
+        dir('centreon-widget-host-monitoring') {
+          checkout scm
+        }
+        sh 'docker run -i --entrypoint "/src/centreon-widget-host-monitoring/ci/scripts/widget-deb-package.sh" -w "/src" -v "$PWD:/src" -e "DISTRIB=bullseye" -e "VERSION=$VERSION" -e "RELEASE=$RELEASE" registry.centreon.com/centreon-debian11-dependencies:22.10'
```

```suggestion
        sh 'docker run -i --entrypoint "/src/centreon-widget-host-monitoring/ci/scripts/widget-deb-package.sh" -w "/src" -v "$PWD:/src" -e "DISTRIB=bullseye" -e "VERSION=$VERSION" -e "RELEASE=$RELEASE" registry.centreon.com/centreon-debian11-dependencies:22.10'
```
This shouldnt be 22.04 instead of 22.10 ?